lib/frrscript.h

```diff
@@ -135,6 +176,7 @@ const struct prefix * : lua_pushprefix                          \
 
 #define DECODE_ARGS_WITH_STATE(L, value)                                       \
 	_Generic((value), \
+int : lua_decode_int_noop,                                      \
```

Comment:
this should probably be `lua_decode_integer_noop` for consistency with the other int ones.

or change those to just be "int".

Reply:
updated to integer since that's closer to how lua spells it 36cf58c